SPECS/dracut/allow-liveos-overlay-no-user-confirmation-prompt.patch

```diff
@@ -0,0 +1,22 @@
+--- a/modules.d/90dmsquash-live/dmsquash-live-root.sh	2021-05-27 05:34:19.000000000 -0700
```

Comment:
⁠Is this a patch from upstream? or is it your contribution to solve this problem?

Reply:
not from upstream. I've authored this patch to unblock the boot flow.